src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.c

```diff
@@ -28,50 +41,193 @@ const ActorInit Bg_Kin2_Bombwall_InitVars = {
     (ActorFunc)BgKin2Bombwall_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80B6E6F0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000008, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 60, 60, 0, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B6E748[] = {
+Color_RGBA8 primColor = { 210, 210, 210, 255 };
+Color_RGBA8 envColor = { 140, 140, 140, 255 };
+
+Vec3f dustAccel = { 0.0f, 0.33f, 0.0f };
+
+s8 D_80B6E730[] = { -60, -34, -8, 18, 44 }; // Used to generate a random vector to modify the dust position vector.
+s16 scales[] = { 25, 23, 21, 19, 17, 15, 13, 10 }; // Scales for random explosion debris.
```

Comment:
```suggestion
static s16 sScales[] = { 25, 23, 21, 19, 17, 15, 13, 10 }; // Scales for random explosion debris.
```
And all of this data from the primColors down can go above the function that uses it.

(The `static` is required to use generic names like this so there aren't symbol collisions.)

Reply:
Done. Will be pushed with next commit.